Formula/spirv-tools.rb

```diff
@@ -0,0 +1,44 @@
+class SpirvTools < Formula
+  desc "API and commands for processing SPIR-V modules"
+  homepage "https://github.com/KhronosGroup/SPIRV-Tools"
+  url "https://github.com/KhronosGroup/SPIRV-Tools/archive/v2019.2.tar.gz"
+  sha256 "1fde9d2a0df920a401441cd77253fc7b3b9ab0578eabda8caaaceaa6c7638440"
+  depends_on "cmake" => :build
+
+  resource "re2" do
+    # DEPS
```

Comment:
comment isn't needed, if the formula doesn't depend on a resource it should just be removed

Reply:
sorry for confusing comment, I mean the revision number is in the file "./DEPS" here